components/micro-gateway-core/src/main/ballerina/src/gateway/utils/utils.bal

```diff
@@ -931,6 +931,9 @@ function appendMultipleJWTIssuers(http:InboundAuthHandler[] handlers) {
 # + return - auth handlers
 public function getHandlers(string[] appSecurity) returns http:InboundAuthHandler[] {
     http:InboundAuthHandler[] handlers = [];
+    if (isDebugEnabled) {
```

Comment:
No need of the check

Reply:
I added it since there is a string concat operation and an array to string conversion.